ReSharper.FSharp/src/FSharp.Psi.Features/src/Util/PsiUtil.fs

```diff
@@ -146,6 +146,10 @@ let isInlineSpace (node: ITreeNode) =
 let isWhitespace (node: ITreeNode) =
     let tokenType = getTokenType node
     isNotNull tokenType && tokenType.IsWhitespace
+    
+let isEquals (node: ITreeNode) =
```

Comment:
maybe to name it more accurately to not be confused with comparison?

Reply:
I'd personally not add it at all.

Reply:
The other functions were added mostly to be passed to other functions as an argument. The only usage for this function can have the logic inlined:
```fsharp
getTokenType expr.Operator.FirstChild == FSharpTokenType.EQUALS
```

But it definitely needs a comment with a `todo:` to fix the rule.